### DISCUSSION TITLE
Debug in Webstorm

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [vercel/next.js#23580] [Help]
Hi.

Tell me please the current way to start the server devs in debug mode so that breakpoints work

**Comment by popuguytheparrot** [comment]
Up

**Comment by decapo01** [comment]
In phpstorm (and I'm sure other jetbrains ides) you can go to the package.json file and you will see  some run buttons next to the scripts.
![image](https://user-images.githubusercontent.com/6868296/117469920-78bfb080-af13-11eb-969b-ad0f4550a15f.png)
you can right click those and a debug button will pop up.

**Comment by ghost** [comment]
This JetBrains blog might help. [How to Debug with WebStorm](https://blog.jetbrains.com/webstorm/2018/01/how-to-debug-with-webstorm/)
